ner_telhub/widgets/menu_widgets.py

```diff
@@ -76,5 +78,39 @@ def __init__(self, parent: QWidget):
         layout = QVBoxLayout()
         layout.addWidget(table)
         self.setLayout(layout)
-        
+
+
+class ButtonStyles(Enum):
+    """Defines the styles of buttons. When hovering over them, they turn two shades brighter"""
+    # Lighter shades found with https://www.w3schools.com/colors/colors_picker.asp
+
+    DEFAULT = ""
+    RED = """QPushButton {color: white; background-color: #FF5656; border-radius: 4px; padding: 3% 8%;}
+        QPushButton:hover {background-color: #ff8080; color: white;}"""
+    GREEN = """QPushButton {color: white; background-color: #1a8f35; border-radius: 4px; padding: 3% 8%;}
+        QPushButton:hover {background-color: #20ac40; color: white;}"""
+    BLUE = """QPushButton {color: white; background-color: #0693E3; border-radius: 4px; padding: 3% 8%;}
+        QPushButton:hover {background-color: #1fa9f9; color: white;}"""
+    GRAY = """QPushButton {color: white; background-color: #999999; border-radius: 4px; padding: 3% 8%;}
+        QPushButton:hover {background-color: #b3b3b3; color: white;}"""
+
+
+
+class NERButton(QPushButton):
+    """Creates a button using a provided preset button style"""
+
+    def __init__(self, title: str, style="DEFAULT"):
```

Comment:
It can be buggy if we pass in a string for the style (crashes the program if the style doesn't exist). Let's make this input an enum value instead (see the comment about the inner Style class)